core/cap-0034.md

```diff
@@ -448,7 +483,7 @@ consensus protocol.
 Considering the `closeTime`-related changes in this CAP also raised a further
 because the `SIGNED` `StellarValue`, besides the transaction set and close time,
 also contains a set of ledger `upgrades`, we could choose additionally to change
-`upgrades` to come from the single winning `StellarValue` rather than being
+`upgrades` to come from the single externalized `StellarValue` rather than being
```

Comment:
I don't think you mean `externalized` here? (selected StellarValue is not externalized yet at this point). I think simply saying "selected `StellarValue`" or "chosen `StellarValue`" is sufficient here.

Reply:
Fixed (I used "selected").